server/sonar-server/src/main/resources/org/sonar/server/computation/ws/quality_gate_status-example.json

```diff
@@ -0,0 +1,70 @@
+{
+  "qualityGateDetails": {
+    "level": "ERROR",
+    "conditions": [
+      {
+        "metric": "new_coverage",
+        "op": "LT",
+        "period": "1",
+        "warning": "",
```

Comment:
You can't compare data in datastore and date in WS response. The first one is technical, the latter is functional and in API.
The reasons I gave are enough to understand:
* info about period are missing (index+date+mode+param)
* empty values are usually not set in our JSON responses
* can db constants GT/LT/... be used in WS ?